packages/api/tests/modules/stats/stats.spec.ts

```diff
@@ -82,24 +85,58 @@ describe("/stats", () => {
                 .get("/stats/requests/median")
                 .query({ start: YESTERDAY.toString(), end: TODAY.toString() })
                 .set("x-access-token", await getAdminToken())
-                .set('Accept', 'application/json');
-            
+                .set("Accept", "application/json");
+
             expect(actual.statusCode).toBe(500);
             expect(actual.body).toEqual(expected);
-        })
+        });
 
         it("should return error with HTTP status code 401", async () => {
             const ERROR_MESSAGE = "Something went wrong";
             spyGetMedianRequestsOnPeriod.mockImplementationOnce(async () => Promise.reject(new Error(ERROR_MESSAGE)));
-            const expected = {success: false, message:  "JWT does not contain required scope." };
+            const expected = { success: false, message: "JWT does not contain required scope." };
             const actual = await request(g.app)
                 .get("/stats/requests/median")
                 .query({ start: YESTERDAY.toString(), end: TODAY.toString() })
                 .set("x-access-token", await getUserToken())
-                .set('Accept', 'application/json');
-            
+                .set("Accept", "application/json");
+
             expect(actual.statusCode).toBe(401);
             expect(actual.body).toEqual(expected);
-        })
-    })
-})
+        });
+    });
+
+    /*
+    describe("getCumulatedUsersPerMonthByYear", () => {
+        const YEAR = 2022
+        const collection = db.collection<UserDbo>("users");
+        beforeEach(() => {
+            collection.insertMany(userFixture)
+        });
+
+        it("should return data with HTTP status code 200", async () => {
+            const DATA = {
+                January: 1,
+                February: 1,
+                March: 1,
+                April: 2,
+                May: 2,
+                June: 2,
+                July: 2,
+                August: 2,
+                September: 2,
+                October: 2,
+                November: 3,
+                December: 3
+            };
+            const expected = { success: true, data: DATA };
+            const actual = await request(g.app)
+                .get(`/stats/users/monthly/${YEAR}`)
+                .set("x-access-token", await getAdminToken())
+                .set("Accept", "application/json")
+                .expect(200)
+            expect(actual.body).toEqual(expected);
+        });
+    });
+*/
```

Comment:
Ce test plante parce qu'il ne reconnaît pas une requête mongo, mais en postman local ça marche. Comme si le mongo du test n'avait pas la même version que le "vrai" mongo. Ça vous dit qqchose ? Quel est la version en (pré-)prod ?

Reply:
@mxmeunier @VSerain

Reply:
Et si c'est mystérieux je propose de faire un ticket et de merge : le reste est prêt et ce serait dommage de bloquer pour un pb de config de tests

Reply:
Le mongo de tests est un mongo in memory ! Tu peut trouver la version dans le fichier jest-mongodb-config.js à la racine de l'api